### PR TITLE
feat: new product switcher with Traefik Hub

### PR DESCRIPTION
**Pull request by paulocfjunior**
- Closes https://github.com/traefik/hub-issues/issues/477

When in the Traefik Hub doc
Screen Shot 2022-10-25 at 00 49 18

When in other docs:
Screen Shot 2022-10-25 at 00 50 37

### CODE REVIEW

File: material/partials/product-switcher.html [hunk 1]
```diff
@@ -33,22 +35,32 @@
...
           </div>
+          {% endif %} {% if config.product != "traefik-hub" %}
+          <div class="dm-item">
+            <div class="dmi-image hub">
+              <img src="{{ 'assets/images/traefik-hub-logo-white.svg' | url }}" alt="Traefik Hub">
+            </div>
+            <a class="dmi-details" href="https://doc.traefik.io/traefik-hub/">
+              <div class="dmi-title">Traefik Hub</div>
+              <div class="dmi-description">
+                The cloud-native application proxy
```
Comment (paulocfjunior) [inline material/partials/product-switcher.html:46]: @geraldcroes what should we put below the Traefik Hub name in this dropdown menu?

Screen Shot 2022-10-25 at 00 50 37

@gndz07 @ldez please feel free to update this part once we have the definition and proceed with review/merge.

File: src/partials/product-switcher.html [hunk 2]
```diff
@@ -59,27 +66,40 @@
...
+            <div class="dmi-image hub">
+              <img
+                src="{{ 'assets/images/traefik-hub-logo-white.svg' | url }}"
+                alt="Traefik Hub"
+              />
+            </div>
+            <a class="dmi-details" href="https://doc.traefik.io/traefik-hub/">
+              <div class="dmi-title">Traefik Hub</div>
+              <div class="dmi-description">
+                The cloud-native application proxy
```
Comment (paulocfjunior) [inline src/partials/product-switcher.html:80]: @geraldcroes what should we put below the Traefik Hub name in this dropdown menu?

Screen Shot 2022-10-25 at 00 50 37

@gndz07 @ldez please feel free to update this part once we have the definition and proceed with review/merge.

Comment (paulocfjunior) [reply src/partials/product-switcher.html:80]: ℹ️ Don't forget to run `npm run build` after this change and commit the changes in the `material` folder.

**Comment by geraldcroes** [comment]
Hey! Great, the official tagline, according to [the Hub 1.0 GTM deck](https://docs.google.com/presentation/d/1J-MACf_BkXv6pqI-2SgU5yqVTvvQXVan1vVIj6kc_-Q/edit#slide=id.g15969115f04_0_5), is 

"Your Cloud Native Networking Platform"

To be consistent with other taglines in the website, I propose we change "Your" with "The", and that we update the case accordingly

So "The cloud native networking platform"

Can you also use this opportunity to remove the hyphenated cloud-native from other tag lines? (according to our style guide, this should never be hyphenated)

**Comment by ldez** [comment]
As I don't have the right to edit your PR (I don't know why), I merge the PR manually 22c3039e7924db34c72261d14a4b81cd99642582

**Pull request closed without merging**